README.rst

```diff
@@ -79,16 +79,20 @@ Project Status
 
 |Documentation Status| |Gitmate|
 
-Getting Involved
-----------------
+Newcomers Guide and Getting Involved
+------------------------------------
 
-If you want to contribute to coala, please take a look at the `Getting
-Involved Information
-<http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__.
+coala is newcomer friendly and we have built an entire website just
+to get any contribution done really easily.
```

Comment:
Hi,

I took the freedom and deleted the cluttering GitMate messages.

I find this "and we have build an entire website just to get any contribution done really easily" not quite satisfying:

1. It's just a documentation page, not an "entire website".
2. It sounds a bit like "we've done something great! Now be grateful! And thank us! Send us chocolate!"
3. "really easily" doesn't sound quite right to me.

(I know that this was phrased by @Adrianzatreanu here but that doesn't change that IMO it shouldn't be accepted. FWIW some contributions take longer, most others had better defined tasks, some had like a typo or something so even across newcomer issues there's some differences.)